.circleci/config.yml

```diff
@@ -173,3 +208,6 @@ workflows:
       - build_cu102:
           requires:
             - build_py3.8_pt1.9_cpu
+      - build_windows:
```

Comment:
is build_windows cheaper than build_cu102? If so shall we consider make build_cu102 requires the success of build_windows?

Reply:
build_windows (40 credit/min) is cheaper than buld_cu102 (240 credit/min).